src/Umbraco.Web.UI.Client/src/views/propertyeditors/multiurlpicker/multiurlpicker.controller.js

```diff
@@ -70,6 +70,7 @@ function multiUrlPickerController($scope, angularHelper, localizationService, en
         
         $scope.linkPickerOverlay = {
             view: "linkpicker",
+            useNodeName: true,
```

Comment:
I've added this setting to the link picker for backwards compatibility, as other property editors/plug-ins might use this overlay and don't expect an empty `name` back (and know to use the `nodeName` instead).